packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -110,11 +113,15 @@ describe('RPC Server Acceptance Tests', function() {
         // get contract details
         mirrorContract = await mirrorNode.get(`/contracts/${contractId}`);
 
+        logger.info('Create basic contract');
+        basicContractReceipt = await servicesNode.deployContract(basicContractJson, process.env.OPERATOR_KEY_MAIN);
```

Comment:
Why do we need to pass the operator key? `servicesNode` contains a fully working client.